diff --git a/src/ofxstatement/tool.py b/src/ofxstatement/tool.py
index 8a3f51c..87c7721 100644
--- a/src/ofxstatement/tool.py
+++ b/src/ofxstatement/tool.py
@@ -15,7 +15,10 @@ log = logging.getLogger(__name__)
 
 
 def get_version():
-    dist = pkg_resources.get_distribution("ofxstatement")
+    try:
+        dist = pkg_resources.get_distribution("ofxstatement")
+    except pkg_resources.DistributionNotFound:
+        return None
     return dist.version
 
 
